data/map.txt

```diff
@@ -27195,7 +27195,7 @@ planet "Fek Rembatu"
 
 planet "Fenrir Station"
 	attributes deep station
-	landscape land/sivael1
+	landscape land/station4
```

Comment:
Station4 fell out of use because it includes people in it, which goes against the landscape image guidelines. We might as well replace it.

Reply:
I tried replacing it with something else. Is the new image alright?